docs/features/index.rst

```diff
@@ -0,0 +1,26 @@
+Features
+========
+
+.. toctree::
+   :maxdepth: 1
+
+   commands
```

Comment:
For a lengthy toctree like this, we should consider alphabetizing to make it easier to visually parse

Reply:
Agreed. Alphabetic sorting, and maybe also categorizing. With a short description of each feature in the table of contents.

Reply:
I pushed a change to make it alphabetical.  Categorizing would be useful as well.